tests/main/snap-confine-from-core/task.yaml

```diff
@@ -38,7 +38,16 @@ execute: |
     test-snapd-tools.echo hello
 
 
-
+    echo "Check if snap-confine profile generation test is applicable"
+    if [ $(snap debug confinement) != "strict" ]; then
```

Comment:
Instead of doing this check please do the check that is equivalent to what is in the code. Run `aa-enabled` and check if it prints `Yes`. Snapd also loads snap-confine-from-core profile on partial apparmor support.